versioned/persist/tx/src/main/java/org/projectnessie/versioned/persist/tx/TxDatabaseAdapter.java

```diff
@@ -1673,7 +1672,7 @@ private RefLogEntry writeRefLogEntry(
       NamedRef ref,
       RefLogHead refLogHead,
       Hash commitHash,
-      Operation operation,
+      RefLogEntry.Operation operation,
```

Comment:
should we prefer `Operation` in this file instead?

Reply:
I think in this particular case using the longer name is justified to identify this as belonging to the deprecated ref log functionality.